dds/DCPS/TypeSupportImpl.h

```diff
@@ -25,6 +25,10 @@ OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 
 
 namespace OpenDDS {
+namespace XTypes{
+class TypeLookupService;
```

Comment:
```suggestion
namespace XTypes {
  class TypeLookupService;
```

Reply:
Looks like there is no indentation inside namespaces in this file.

Reply:
It's described in the guidelines.  Not a big deal though...

Reply:
Okay, I see that in the guidelines and have updated this. I think we may add to the rule to say that it applies to the innermost namespace scope.